azure-pipelines.yml

```
@@ -69,14 +69,9 @@ stages:
             steps:
               - checkout: self
                 clean: true
-              - powershell: |
-                  SqlLocalDB stop
-                  SqlLocalDB delete
-                  Remove-Item "$env:USERPROFILE\*.mdf"
-                  Remove-Item "$env:USERPROFILE\*.ldf"
-                  SqlLocalDB create
-                  SqlLocalDB start
-                displayName: Cleanup databases
+              - powershell: Get-ChildItem -Recurse -ErrorAction SilentlyContinue 'C:\Program Files*\sqlcmd.exe'
```

```suggestion
              - powershell: Get-ChildItem "C:\Program Files\Microsoft SQL Server\*\Tools\Binn\sqlcmd.exe",
                  "C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\*\Tools\Binn\sqlcmd.exe" |
                Sort-Object -Property @{Expression={$_.FullName.Length}; Descending=$true},
                  @{Expression={$_.FullName}; Descending=$true} |
                Select-Object -First 1 -Property FullName |
                Foreach-Object {"##vso[task.prependpath]$_"}
```
This should be fairly resilient if new SQL Server versions are added though it may be wrong to prefer Client SDK over all regular SQL Server installations.